templates/plugin/memcached.conf.erb

```diff
@@ -1,9 +1,16 @@
 <Plugin memcached>
```

Comment:
hey @mrunge, are you interested in turning this into an epp template?

Reply:
@mrunge We prefer epp for new templates going forward, but will merge this for now.